data/en/test_average_equal_or_less_validation_against_total.json

```diff
@@ -0,0 +1,93 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "survey_id": "0",
+    "title": "Grouped validation against total test survey",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "A survey that tests the average of breakdown question answers against a total",
+    "sections": [{
+        "id": "default-section",
+        "groups": [{
+            "id": "group",
+            "title": "Validate average against total",
+            "blocks": [{
+                    "type": "Question",
+                    "title": "Target Total",
+                    "id": "total-block",
+                    "description": "",
+                    "questions": [{
+                        "id": "total-question",
+                        "title": "Total",
+                        "type": "General",
+                        "answers": [{
+                            "id": "total-answer",
+                            "label": "Total",
+                            "mandatory": true,
+                            "type": "Number",
+                            "decimal_places": 2,
+                            "max_value": {
+                                "value": 100
+                            }
+                        }]
+                    }]
+                },
+                {
+                    "type": "Question",
+                    "title": "Calculated Total",
+                    "id": "breakdown-block",
+                    "questions": [{
+                        "id": "breakdown-question",
+                        "title": "Breakdown",
+                        "type": "Calculated",
+                        "calculated": {
+                            "calculated_type": "average",
+                            "answer_id": "total-answer",
+                            "answers_to_group": [
```

Comment:
I think using the word "group" should be avoided in here. 1) we aren't actually grouping the answers and 2) we've already given the word "group" a meaning within the application.

Perhaps change to "answers_to_aggregate"?

Reply:
how about `answers_to_calculate`?

Reply:
Nice - let's go with that.

Reply:
I will need to raise a PR in schema validator for this too as the change has been merged earlier